### PR TITLE
disable active storage and fix icon

### PR DESCRIPTION
**Pull request by rueyaa332266**
Fix the problem by disable active storage when using RAILS_ENV production.

Fix the problem that overwriting gemfile.log when mounting the volume.

Fix the logo also.

### CODE REVIEW

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+version: '3'
```
Comment (naotospace) [inline docker-compose.dev.yml:1]: Thanks for fixing!

I will show my idea below.

1. Fix docker-compose.dev.yml
```yaml
version: '3'
services:
  app:
    build:
      args:
        RAILS_ENV: ${RAILS_ENV:-development}
    volumes:
      - .:/app
    environment:
      - RAILS_ENV=${RAILS_ENV:-development}
      - BUCKY_DB_NAME=${BUCKY_DB_NAME:-bucky_development}
 ```

2. Add docker-compose up procedures for development in README.md
```md
COMPOSE_FILE=docker-compose.yml:docker-compose.dev.yml docker-compose up -d
or
docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d
```

**Review by naotospace — COMMENTED** [review]
Thanks for PR.
I added a comment.
PLS check it.

File: docker-compose.dev.yml (hunk 1 above)
Comment (naotospace) [reply docker-compose.dev.yml:1]: Maybe You should fix docker-compose.without_db.yml also.

Comment (rueyaa332266) [reply docker-compose.dev.yml:1]: Thanks,  I fixed compose files and README!

File: README.md [hunk 2]
```diff
@@ -6,49 +6,65 @@
 Bucky-management is a web application that shows test result executed by [Bucky-core](https://github.com/lifull-dev/bucky-core).
 
 ## Getting Started
-We prepare two docker-compose files to start up Bucky-managemnt.
-- **docker-compose.yml**: Using DB container in docker-compose.yml
-- **docker-compose.without_db.yml**: Using external DB
+We prepare three docker-compose files to start up Bucky-managemnt.
+- **docker-compose.yml**: Base compose file for production
+- **docker-compose.with_nmysql.yml**: Compose file for adding a DB container
```
Comment (kishinao) [inline README.md:11]:
```suggestion
- **docker-compose.with_mysql.yml**: Compose file for adding a DB container
```

**Review by kishinao — CHANGES_REQUESTED** [review]
Thanks! Almost LGTM.
But I found a typo. PLS check it.

File: README.md (hunk 2 above)
Comment (rueyaa332266) [reply README.md]: Thanks! I fix it.

**Review by naotospace — APPROVED** [review]
LGTM!

**Review by kishinao — APPROVED** [review]

**Pull request merged**